src/theme.css

```diff
@@ -36,12 +36,12 @@
   --ion-color-primary-shade: #3171e0;
   --ion-color-primary-tint: #4c8dff;
 
-  --ion-color-secondary: #0cd1e8;
-  --ion-color-secondary-rgb: 12, 209, 232;
+  --ion-color-secondary: #34495e;
+  --ion-color-secondary-rgb: 52,73,94;
   --ion-color-secondary-contrast: #ffffff;
-  --ion-color-secondary-contrast-rgb: 255, 255, 255;
-  --ion-color-secondary-shade: #0bb8cc;
-  --ion-color-secondary-tint: #24d6ea;
+  --ion-color-secondary-contrast-rgb: 255,255,255;
+  --ion-color-secondary-shade: #2e4053;
+  --ion-color-secondary-tint: #485b6e;
```

Comment:
I like that you found those guys. Would it be possible for you to replace "orange" and different colors to variables for the entire proejct? (skip if you already did)